test/evergreen.yml

```diff
@@ -1798,6 +1823,30 @@ tasks:
             set -o verbose
             for i in {1..10}; do ${python_binary|python} split_stress.py; done
 
+  - name: format-stress-test
+    # Set 25 hours timeout
+    exec_timeout_secs: 90000
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger with builtins"
+      - func: "format test script"
+        vars:
+          #run for 24 hours ( 24 * 60 = 1440 minutes), use default config
+          format_test_script_args: -b "SEGFAULT_SIGNALS=all catchsegv ./t" -t 1440
+
+  - name: format-stress-smoke-test
+    # Set 25 hours timeout
+    exec_timeout_secs: 90000
```

Comment:
I believe this smoke test won't be running up to 24 hrs, so the timeout setting here feels too much. Please refer to #5032 for a similar change.

Reply:
@lukech I have already merges my changes. @raviprakashgiri29 Please pull latest changes from `develop` and merge the changes.

Reply:
Fixed